dynamiq/storages/vector/milvus/milvus.py

```diff
@@ -25,6 +30,8 @@ def __init__(
         connection: Milvus | None = None,
         client: Optional["MilvusClient"] = None,
         index_name: str = "default",
+        content_key: str = "content",
```

Comment:
Please add new params at the end to avoid breaking changes as users could pass params without keys.